lib/tool_shed/galaxy_install/tool_dependencies/recipe/tag_handler.py

```diff
@@ -418,9 +418,9 @@ def handle_complex_repository_dependency_for_package( self, elem, package_name,
                                                                                                         dependent_install_dir,
                                                                                                         tool_dependency_type='package' )
                     if not can_install_tool_dependency:
-                        log.debug( "Tool dependency %s version %s cannot be installed (it was probably previously installed), " %
-                            ( str( tool_dependency.name, str( tool_dependency.version ) ) ) )
-                        log.debug( "so appending it to the list of handled tool dependencies." )
+                        log.debug( "Tool dependency %s version %s cannot be installed (it was probably previously installed), "
+                                   "so appending it to the list of handled tool dependencies.",
+                                   str( tool_dependency.name), str( tool_dependency.version ) )
```

Comment:
Not necessary using the log.<level> functions, string params are passed as extra arguments.